model/accounts_mgmt/v1/notify_details_request_type.model

```diff
@@ -37,4 +37,7 @@ struct NotificationDetailsRequest {
 
 	// Indicates whether the service log is internal only to RH.
 	InternalOnly Boolean
+
+	// Indicates the type of the service log.
+    LogType String
```

Comment:
is this actually a string or an enum type erased to a string? not sure if the meta-model supports specifying valid values, but if so we should do that. basically we want as much information here as we would want in openapi docs or a generated client to use the api correctly without needing to go to a secondary source or person for clarification. good descriptions, allowed values, etc.

maybe thats something we come back to in the future in our quest for a single source of truth for API definitions...